doc/conf.py

```diff
@@ -41,16 +41,28 @@
 
 # General information about the project.
 project = u'screed'
-copyright = u'2012, Alex Nolley and Titus Brown'
+copyright = u'2012-2015, Michigan State University'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = '0.7'
+
 # The full version, including alpha/beta/rc tags.
-release = '0.7.1'
+
+sys.path.insert(0, '.')
+import versioneer
```

Comment:
Might be wise to use the same hack around the current versioneer import bug that `setup.py` uses: https://github.com/ged-lab/screed/blob/master/setup.py#L8